cerberus/client.py

```diff
@@ -76,8 +76,13 @@ def _set_token(self):
         except:
             pass
         if self.username:
-            ua = UserAuth(self.cerberus_url, self.username, self.password)
-            self.token = ua.get_token()
+            try:
+                ua = UserAuth(self.cerberus_url, self.username, self.password)
+                self.token = ua.get_token()
+            except CerberusClientException as e:
+                print("Error: " + str(e))
+            except Exception as e:
+                print("Error: " + str(e))
```

Comment:
I think there might be a problem with the flow. If I understand this correctly, when `CerberusClientException` or `Exception` gets thrown by `UserAuth.get_token`, this code prints the error message and moves on. But then `self.token` remains `None` because  `UserAuth.get_token` failed. I think it's better to throw out the exception when `UserAuth.get_token` fails than to wait for subsequent operations (read secret) to fail caused by empty token.

Reply:
Ah.  That is problematic.
Should I just remove the `try` `except` block, and let the errors just raise on through and let whoever is using the library deal with the CerberusClientException?

I guess the only reason that try except was there was mainly for quick testing.  So I'm pretty OK with removing that logic.

Reply:
I agree that removing the `try except` is the way to go.

Reply:
Change pushed!